openmicroscopy/bioformats
# Add support for Overlays (ImageJ)

To Test : 

1) Open an Image in ImageJ.
2) Draw ROI's , and this time draw regions using IJ toolbar and Image > Overlay > Add selection or use the shortcut cmd B. 
(This adds the the roi's as an overlay to the Image. (Overlays are non-active selections displayed ‘over’ the pixel data,In a way you can think of the image overlay as an invisible ROI Manager↓ in which selections are being added, allowing ROIs to be on ‘hold’)
3) export the image using Plugins > Bio-formats > Bio-formats Exporter to the local drive on your machine
Note that any regions in ROI Manager will be ignored in case the image has an Overlay to it.
4) Reimport the Image into ImageJ using Plugins > Bio-formats > Bio-formats Importer, with display Roi's checkbox selected. 
5) Check if the ROI's are getting populated in the ImageJ ROI Manager.
6) Check that when you tick the Show all tickbox in ROI Manager you see the regions just as you saved them on the image
